src/consensus/vote.cpp

```diff
@@ -100,6 +105,7 @@ VoteManager::~VoteManager() { daemon_->join(); }
 void VoteManager::setNetwork(std::weak_ptr<Network> network) { network_ = move(network); }
 
 void VoteManager::retreieveVotes_() {
+  LOG(log_si_) << "Retrieve unverified votes from DB";
```

Comment:
are you sure about this level?

Reply:
yeah, since retreieveVotes_ only run in the beginning of node start. The log only print one time per each reboot. I set it in log_si_ for checking the thread is running in the beginning